code.json

```diff
@@ -91,22 +91,22 @@
     "name": "ISIS3",
     "organization": "U.S. Geological Survey",
     "description": "GitHub code repository for the Integrated Software for Imagers and Spectrometers (ISIS)",
-    "version": "8.1.0_RC2",
+    "version": "8.3.0_RC1",
     "status": "Production",
 
     "permissions": {
       "usageType": "openSource",
       "licenses": [
         {
           "name": "Public Domain, CC0-1.0",
-          "URL": "https://code.usgs.gov/astrogeology/isis/-/raw/8.1.0_RC2/LICENSE.md"
+          "URL": "https://code.usgs.gov/astrogeology/isis/-/raw/8.3/LICENSE.md"
```

Comment:
Are we leaving the urls (this one and below urls) to use version `8.3` when the full current version is `8.3.0_RC1`?